### PR TITLE
fix: [CDS-36578]: helm command flag field updated to monaco editor

### PR DESCRIPTION
**Pull request by bhavya-sinha**
##### Screenshots:
Screenshot 2022-04-26 at 2 36 28 PM
Screenshot 2022-04-26 at 2 39 20 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by bhavya-sinha** [comment]
retrigger build

**Comment by bhavya-sinha** [comment]
trigger cypress

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmAdvancedStepSection.tsx [hunk 1]
```diff
@@ -122,43 +123,50 @@ function HelmAdvancedStepSection({
...
-                                onChange={value =>
-                                  formik.setFieldValue(`formik.values?.commandFlags?.[${index}].flag`, value)
-                                }
-                                isReadonly={isReadonly}
+                            label={index === 0 ? getString('flag') : ''}
+                            allowedTypes={allowableTypes}
+                            style={{ width: 450 }}
+                            skipRenderValueInExpressionLabel
+                            expressionRender={() => (
+                              <MonacoTextField
```
Comment (chiragp1212) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmAdvancedStepSection.tsx:152]: we can memoise this component and use the same in line 152 and 161

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithGIT/HelmWithGIT.module.scss [hunk 2]
```diff
@@ -21,10 +21,30 @@
...
+      width: 240px;
+      margin-right: var(--spacing-xxlarge);
+      :global {
+        .bp3-label {
+          margin: var(--spacing-small) 0px;
+        }
+      }
+    }
+    .commandFlagType {
+      margin-top: 31px;
```
Comment (chiragp1212) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithGIT/HelmWithGIT.module.scss:40]: generally we deal in multiples of 8px

**Review by chiragp1212 — APPROVED** [review]

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithGIT/HelmWithGIT.module.scss (hunk 2 above)
Comment (bhavya-sinha) [reply src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithGIT/HelmWithGIT.module.scss:40]: But 30px was not looking good, so had to do this

**Review by chetanvekariya-harness — APPROVED** [review]

**Pull request merged**